ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,39 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.3.0-beta2 4-Oct-2022
+Changes from 4.3.0-beta1
+1) Autopilot specific fixes and enhancements
+    a) AIRLink autopilot supports UART2
+    b) CUAV V6X supports CAN battery monitor by default
+    c) MatekF405-CAN board uses less memory to fix compass calibration issues
```

Comment:
I think this was CAN in general using less memory so that calibration works on the MatekF405-CAN

Reply:
no, it only affected one board, in hwdef.dat

Reply:
You beat me to it, I was about to remove my comment :D